README.rst

```diff
@@ -32,21 +32,49 @@ Javascript. With Batavia, you can run Python bytecode in your browser.
 It honors Python 3.4.4+ syntax and conventions, and let's you
 reference objects and classes defined natively in JavaScript.
 
-Tutorial
---------
+Quick Start
+---------------
 
-To take Batavia for a spin, run through the `Getting Started guide`_.
+(For a more detailed setup, please see the `Getting Started tutorial <https://batavia.readthedocs.io/en/latest/tutorials/tutorial-0.html>`_)
```

Comment:
Not sure whether starting a new section with a "see more" parentheses is best practice, usually this info should go at the end of the section as a regular paragraph. Alternatively, you can state the purpose of this section "Here's a high-level overview of what you need to do", followed by this sentence as a regular sentence in a paragraph.